packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -647,6 +647,27 @@ addFieldsDict(Posts, {
     group: formGroups.event,
     hidden: (props) => !props.eventForm,
   },
+  
+  eventType: {
+    type: String,
+    viewableBy: ['guests'],
+    insertableBy: ['members'],
+    editableBy: ['members'],
+    hidden: (props) => !props.eventForm,
+    control: 'select',
+    group: formGroups.event,
+    optional: true,
+    order: 2,
+    form: {
+      options: [
+        {value: 'presentation', label: 'Presentation'},
+        {value: 'discussion', label: 'Discussion'},
+        {value: 'workshop', label: 'Workshop'},
+        {value: 'social', label: 'Social'},
+        {value: 'coworking', label: 'Coworking'},
```

Comment:
I tried to make this list useful by covering most kinds of events and having very little overlap between them (ex. I used to have "book club" but that felt similar enough to "discussion" that I decided to remove it). Please let me know if you have any suggested changes here.

Reply:
Maybe another type is class/course? E.g. https://forum.effectivealtruism.org/events/HpbJbdDfcYDkBHadm/ea-for-christians-intro-course

I'm not sure if this is different enough from "presentation" to justify the different label.